cli/flags.rs

```diff
@@ -65,6 +65,8 @@ pub enum DenoSubcommand {
   },
   Test {
     fail_fast: bool,
+    report_to_console: bool,
+    disable_log: bool,
```

Comment:
No need to specify these two field here - they are internal to `Deno.runTests()` (which will be made private anyway); use `quiet` instead

Reply:
merged into quiet 👍